chains/evm/voter/voter.go

```diff
@@ -156,7 +157,27 @@ func (v *EVMVoter) shouldVoteForProposal(prop *proposal.Proposal, tries int) (bo
 		return v.shouldVoteForProposal(prop, tries)
 	}
 
-	return true, nil
+	return v.simulateCall(prop, 0)
+}
+
+func (v *EVMVoter) simulateCall(prop *proposal.Proposal, tries int) (bool, error) {
+	Sleep(time.Duration(rand.Intn(shouldVoteCheckPeriod)) * time.Second)
```

Comment:
Don't think you need this. By default it should just try to simulate call and if it fails it can have some sleep.